ls1intum/Artemis
# `Development`: Update server dependencies to Spring Boot 3, Spring Framework 6, Spring Cloud 4, Hibernate 6

# CURRENTLY BLOCKED




### Checklist


### Motivation and Context
We should stay up to date.

### Description


### Steps for Testing
1. Developers should test the development setup, starting the server and client application and communicating 
2. Testers should click through most views of the Artemis application and review if you notice issues while testing different CRUD functionality


### Review Progress






### Test Coverage
no changes

### Screenshots
no UI changes
